cmd/csi-provisioner/csi-provisioner.go

```diff
@@ -44,11 +44,25 @@ import (
 	utilflag "k8s.io/apiserver/pkg/util/flag"
 )
 
+// deprecatedOption can be used to define a deprecated command line
+// flag. Whenever it is set, regardless to which value, a warning is
+// triggered.
+type deprecatedOption struct {
+	option string
+}
+
+func (d deprecatedOption) String() string { return "" }
+func (d deprecatedOption) Set(value string) error {
```

Comment:
could you be explicit in what interface this is implementing for strict interface checking (mostly for code maintainability and my understanding)
e.g:
`var _ InterfaceName := &deprecatedOption{}`